<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQTT Utility Beta v1.1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="newConnButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border:0px}QPushButton:hover{background-color:rgba(44,163,225,50)}QPushButton:pressed{background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/image/add.gif</normaloff>:/image/add.gif</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="authButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{border:0px}QPushButton:hover{background-color:rgba(44,163,225,50)}QPushButton:pressed{background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/image/auth.gif</normaloff>:/image/auth.gif</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>669</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="serverButtons">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>291</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="serversLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="res.qrc">:/image/connection.gif</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>MQTT Connection</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>255</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="connWidget">
         <property name="styleSheet">
          <string notr="true">QTabWidget:pane{border:1px solid rgba(0,0,0,100);background:transparent;}</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew_connect"/>
    <addaction name="authAction"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew_connect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/add.gif</normaloff>:/image/add.gif</iconset>
   </property>
   <property name="text">
    <string>New Connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="authAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/auth.gif</normaloff>:/image/auth.gif</iconset>
   </property>
   <property name="text">
    <string>New Auth Connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
